src/components/HomeChurchContactModal/homeChurchContactModal.scss

```diff
@@ -48,10 +48,10 @@
             padding: 0.75rem 1.25rem;
             white-space: nowrap;
             &:hover {
```

Comment:
The quotation marks make this property invalid.

Also, I don't think this `:hover` selector is doing anything since the React Bootstrap styles need to be overridden. Perhaps just remove these lines?